src/webapi/element.rs

```diff
@@ -266,6 +268,19 @@ pub trait IElement: INode + IParentNode + IChildNode + ISlotable {
             ).into_reference_unchecked()
         }
     }
+
+    /// Request this element and its children be made fullscreen
+    ///
+    /// Note: this may only be called during a user interaction.
+    /// Not all elements may be full-screened, see JS docs for details.
+    /// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/API/Element/requestFullscreen)
+    // https://fullscreen.spec.whatwg.org/#dom-element-requestfullscreen
+    fn request_fullscreen( &self ) -> TypedPromise<(), TypeError> {
```

Comment:
Same as the other one, attach `#[cfg(feature = "experimental_features_which_may_break_on_minor_version_bumps")]` to it.